### PR TITLE
Fix tag checking when adding tags on report bug page

### PR DESCRIPTION
**Pull request by atrol**
Fixes #24632

### CODE REVIEW

File: js/common.js [hunk 1]
```diff
@@ -355,7 +355,8 @@ $(document).ready( function() {
 		var currentTagString = $('#tag_string').val();
 		var newTagOptionID = $(this).val();
 		var newTag = $('#tag_select option[value=' + newTagOptionID + ']').text();
-		if (currentTagString.indexOf(newTag) == -1) {
+		var tagArray = currentTagString.split(tagSeparator);
+		if (tagArray.indexOf(newTag) == -1) {
```
Comment (vboctor) [inline js/common.js]: Spacing is incorrect.

Comment (atrol) [reply js/common.js]: @vboctor please explain. I don't see what's wrong with it.
e.g., isn't the spacing of the 2nd line the same as before? Just replaced `currentTagString` by `tagArray`

Comment (vboctor) [reply js/common.js]: @atrol It is not about the indentation, but about the spacing around if statement and parenthesis.  See below fixed example:

```
		var tagArray = currentTagString.split( tagSeparator );
		if( tagArray.indexOf( newTag ) == -1 ) {
```

Comment (atrol) [reply js/common.js]: I just used the style that I found around those lines. 
AFAIK we don't have Coding Guidelines for JavaScript.
@vboctor so we should use the PHP Coding Guidelines, at least for spacing rules?

Comment (dregad) [reply js/common.js]: > so we should use the PHP Coding Guidelines [for JavaScript], at least for spacing rules?

I don't think we should.

**Pull request merged**